scripts/configure.py

```diff
@@ -994,6 +1016,13 @@ def print_progress(tasks):
       tasks += print_progress(_stop_service('amplipi-updater-test'))
       tasks += print_progress(_remove_service('amplipi-updater-test'))
 
+  # bring up amplipi-tasks
+  tasks += print_progress(_create_service('amplipi-tasks', _tasks_service(env['base_dir']), env))
+  tasks += print_progress(_enable_service('amplipi-tasks'))
+  if not env['is_ci']:
+    tasks += print_progress(_restart_service('redis-server'))  # TODO: is this needed?
```

Comment:
> is this necessary?

Kinda a complicated question. Strictly necessary? nope. but what if the form & function of the way we use redis changes? restarting is a simple way to wipe everything out, unless you have the on disk persistence enabled (I expect that it is disabled, but it is probably worth checking.) This will also act as a bit of a forcing function for us to never use this as a persistent data store, which I have seen folks do and seems unwise without further justification. With a little thought, I'm in favor of keeping a restart here.

Reply:
though I guess a restart of the appliance as a whole probably accomplishes this too :thinking: